app/config/FrontendAppConfig.scala

```diff
@@ -84,8 +84,12 @@ class FrontendAppConfig @Inject()(configuration: Configuration, servicesConfig:
   def sessionCacheUrl(sessionId: String, linkId: String): String =
     customsFinancialsSessionCacheUrl + s"/account-link/$sessionId/$linkId"
 
-  def deleteNotificationUrl(fileRole: FileRole, eori: String): String =
-    customsFinancialsApi + s"/eori/$eori/requested-notifications/$fileRole"
+  def deleteNotificationUrl(fileRole: FileRole, eori: String): String = {
+    fileRole match {
+      case CashStatement => customsFinancialsApi + s"/eori/$eori/notifications/$fileRole"
```

Comment:
Could you please make use of interpolation here and line 90 as well?

Reply:
@tyagikumarvineet updated line 89 and 90 to use interpolation - s"$customsFinancialsApi/eori/$eori/notifications/$fileRole"